app/src/main/kotlin/com/wire/android/ui/common/avatar/UnreadInfoIndicator.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.common.avatar
+
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import com.wire.android.R
+import com.wire.android.util.ui.PreviewMultipleThemes
+
+@Composable
+fun UnreadInfoIndicator(
+    modifier: Modifier = Modifier
+) {
+    Icon(
+        painter = painterResource(id = R.drawable.ic_info_unread),
+        contentDescription = null,
```

Comment:
Suggestion: Since we are already taking care of some accessibility debt, it would be nice to have this one here since the beginning. 

Might be better in the component that uses this icon, but can not tell for sure, either way works :D

Reply:
Not sure if we have make it accessible for talkback, it's a small item and not even clickable

Reply:
Makes sense, let's see what product or maybe Astrid says, either way is fine to me :D